src/web/components/Donut.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { headline } from '@guardian/src-foundations/typography';
+import { text } from '@guardian/src-foundations/palette';
+
+type Props = {
+    sections: Section[];
+    percentCutout?: number;
+    width?: number;
+    height?: number;
+};
+
+type Section = {
+    label: string;
+    value: number;
+    color: string;
+};
+
+const unitStyles = css`
+    ${headline.medium({ fontWeight: 'bold' })}
+    text-anchor: middle;
+`;
+
+const valueStyles = css`
+    ${headline.small({ fontWeight: 'bold' })}
+    fill: ${text.ctaPrimary};
+    text-anchor: middle;
+`;
+
+const labelStyles = css`
+    ${headline.xxsmall()}
+    fill: ${text.ctaPrimary};
+    text-anchor: middle;
+`;
+
+const withoutZeroSections = (sections: Section[]) =>
+    sections.filter(section => section.value !== 0);
+
+export const Donut = ({
+    sections,
+    percentCutout = 35,
+    width = 300,
+    height = 300,
+}: Props) => {
+    if (withoutZeroSections(sections).length === 1) {
+        console.warn('The Donut component requires at least 2 sections');
+        // TODO: Support showing 100% for a single section
+        return null;
+    }
+
+    if (sections.find(section => section.value === 0)) {
+        console.warn(
```

Comment:
github doesnt seem happy with `console.warn`, maybe you can add a eslint ignore?

Reply:
👍 Removed